templates/es/objection-default.txt

```diff
@@ -0,0 +1,15 @@
+A quien corresponda:
+
+Por la presente, me opongo al procesamiento de mis datos personales con fines de marketing directo de acuerdo con el artículo 21 (2) de la GDPR.  Esta oposición incluyte también su uso para la elaboración de perfiles en la medida en que esté relacionada con el marketing directo.
+
+Si hubiera dado mi consentimiento para el procesamiento de mis datos personales con fines de marketing directo (por ejemplo, de acuerdo con el Art. 6(1) o el Art. 9(2) GDPR), por la presente revoco dicho consentiemiento.
```

Comment:
* `GDPR` -> `RGPD`

optional:
* `por ejemplo` -> `p.ej.` (for consistency)
* `Art. 6(1) o el Art. 9(2)` -> `artículo 6(1) o el artículo 9(2) del` (for consistency)